modules/appData.js

```diff
@@ -0,0 +1,3 @@
+const appId = 'PVsyyzJcjAI3kwRWNz7v';
```

Comment:
- [ ] It will be great to ensure that you don't expose the AppID. This is sensitive information about the API and it shouldn't be committed to GitHub. This ensures security in your application. Just provide instructions on how to obtain the keys needed to use the API. Or use `env` to store your appID.

Reply:
- [ ] As previously requested by the reviewer, Please do not 
commit confidential information about the API to GitHub. Provide instructions on how to obtain the keys needed to use the API or use the .env to store confidential data 👍🏾